Project3/styles.css

```
@@ -0,0 +1,134 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+/* main styles*/
+
+body {
+    background-color: #DCE7EA;
+    font-family: 'Noto Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+
+}
+
+header {
+    background-color: #B74926;
+    font-family: 'Noto Sans', sans-serif;
+    margin: 15px;
+    min-width: 375px;
+
+    padding: 10px;
+    position: relative;
+    border-bottom-style: double;
+
+
+
+}
+
+h1 {
+    text-align: center;
+    font-size: 35px;
+}
+
+.container {
+    text-align: center;
+}
+
+.timelineitem {
+    display: inline-block;
+    height: 300px;
+    margin: 10px;
+    position: relative;
+    width: 300px;
+}
+
+.imgbox {
+    overflow: hidden;
+    position: relative;
+}
+
+img {
+    max-width: 100%;
+    transition: transform 2s;
+
+}
+
+/* hover effect and animation */
+
+.container .timelineitem:hover .imgbox img {
```

Probably don't need this overly specific classname?
```suggestion
.timelineitem:hover img {
```